go/interpreter/lfvm/gas_test.go

```diff
@@ -59,3 +59,17 @@ func TestGas_CallGasCalculation(t *testing.T) {
 		})
 	}
 }
+
+func TestGas_CheckGasCostsForAllExecutableOpcodes(t *testing.T) {
+	for i := 0; i < int(NUM_OPCODES); i++ {
+
+		if static_gas_prices[i] == UNKNOWN_GAS_PRICE &&
+			op_2_op[i] != INVALID {
+			t.Errorf("gas price for %v is unknown", OpCode(i))
+		}
+		if static_gas_prices_berlin[i] == UNKNOWN_GAS_PRICE &&
```

Comment:
Since the test looks so similar, I am wondering why it is required to have a separate list of gas prices for the Berlin revision?

Reply:
this is only testing that all `INVALID` opcodes have `UNKNOWN_GAS_PRICE`. The two maps are needed because there 10 instructions with a different gas cost.